src/main/scala/org/ergoplatform/mining/mining.scala

```diff
@@ -14,11 +16,31 @@ package object mining {
 
   val group: BcDlogGroup[EcPointType] = CryptoConstants.dlogGroup
 
+  // Group order, used in Autolykos V.1
   val q: BigInt = group.order
 
   private val hashFn: NumericHash = new NumericHash(q)
 
-  def hash(in: Array[Byte]): BigInt = hashFn.hash(in)
+  /**
+    * Hash function which output is in Zq. Used in Autolykos v.1
+    * @param in - input (bitstring)
+    * @return - output(in Zq)
+    */
+  def hashModQ(in: Array[Byte]): BigInt = hashFn.hash(in)
```

Comment:
It is better to leave the old name in this release. Otherwise it is hard to see that the old protocol is preserved with so many changes in this PR.

Reply:
@aslesarenko we're using another hash function in addition to this one now in PoW (v2)